build_run_docker.sh

```diff
@@ -117,6 +119,11 @@ while [[ $# -gt 0 ]]; do
         shift # past argument
         ;;
 
+    -lv | --volumes-in-local-dir)
+        VOLUMES_DIR="$(pwd)/docker_volumes/"
```

Comment:
That's fair, I'll make those changes. Are you removing the volumes between runs?

Reply:
Yeah, you'll find this somewhere in the script, which I always leave uncommented:

```
## Uncomment this for quicker testing
# yes | docker container prune
# yes | docker volume prune
```

Reply:
Pushed.

Reply:
Just double checking - you don't want to remove the local volume dirs between runs? Up to you. You could just add a commented-out command for convenience. I don't run the prune commands by default because it might delete other important unrelated volumes.

Reply:
I added that, on line 392.